ui/lib/core/addon/utils/client-count-utils.ts

```diff
@@ -229,23 +222,6 @@ export const formatByNamespace = (namespaceArray: NamespaceObject[] | null): ByN
   });
 };
 
-export const formatTableData = (byMonthNewClients: ByMonthNewClients[], month: string): TableData[] => {
```

Comment:
this is basically now this https://github.com/hashicorp/vault/pull/31493/files#diff-3df4c9aa8d0c055ff1e929e3fc3051ffaa239eb00a3176e9db11586c722c1b99R265-R266